tests/Microsoft.Identity.Test.Unit.net45/TelemetryTests/RegionalTelemetryTests.cs

```diff
@@ -0,0 +1,254 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.Linq;
+using System.Net.Http;
+using Microsoft.Identity.Test.Unit.Throttling;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Client.OAuth2.Throttling;
+using Microsoft.Identity.Client.TelemetryCore;
+using Microsoft.Identity.Test.Common.Core.Helpers;
+using Microsoft.Identity.Test.Common.Core.Mocks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using static Microsoft.Identity.Client.TelemetryCore.Internal.Events.ApiEvent;
+
+namespace Microsoft.Identity.Test.Unit.TelemetryTests
+{
+    [TestClass]
+    public class RegionalTelemetryTests : TestBase
+    {
+        private MockHttpAndServiceBundle _harness;
+        private ConfidentialClientApplication _app;
```

Comment:
Not sure why these are class variables instead of being passed to the methods that uses these. only the test method initializes these.